Training_Center_Task_10/Tests.cs

```diff
@@ -0,0 +1,301 @@
+            Assert.AreEqual(true, texasSelected);
+
+            _driver.Wait(3000);
+        }
+
+        [Description("Task 6")]
+        [TestMethod]
+        public void AlertTestJavaScriptConfirmBoxVerifyAlertText()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click me!" button
+            IWebElement ClickMeButton = _driver.FindElement(By.CssSelector("button[onclick='myConfirmFunction()']"));
+            ClickMeButton.Click();
+
+            _driver.Wait(2000);
+
+            //Pick alert text
+            string alertText = _driver.SwitchTo().Alert().Text;
+
+            Assert.AreEqual("Press a button!", alertText);
+        }
+
+        [Description("Task 6")]
+        [TestMethod]
+        public void AlertTestJavaScriptConfirmBoxVerifyAlertClickedOK()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click me!" button
+            IWebElement ClickMeButton = _driver.FindElement(By.CssSelector("button[onclick='myConfirmFunction()']"));
+            ClickMeButton.Click();
+
+            _driver.Wait(2000);
+
+            //Cliok OK on alert
+            _driver.SwitchTo().Alert().Accept();
+
+            IWebElement alertClickedOk = _driver.FindElement(By.Id("confirm-demo"), 5);
+
+            Assert.AreEqual("You pressed OK!", alertClickedOk.Text);
+        }
+
+        [Description("Task 6")]
+        [TestMethod]
+        public void AlertTestJavaScriptAlertBoxNameEnteredCorrectly()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+
+            //Click "Click for Prompt Box" button
+            IWebElement ClickForPromptBoxButton = _driver.FindElement(By.CssSelector("button[onclick='myPromptFunction()']"));
+            ClickForPromptBoxButton.Click();
+
+            _driver.Wait(2000);
+
+            //Enter the name in alert and click OK
+            _driver.SwitchTo().Alert().SendKeys("Ivo");
+            _driver.SwitchTo().Alert().Accept();
+
+            IWebElement message = _driver.FindElement(By.Id("prompt-demo"), 5);
+
+            Assert.AreEqual("You have entered 'Ivo' !", message.Text);
+        }
+
+        [Description("Task 7")]
+        [TestMethod]
+        public void WaitForUser()
+        {
+            _driver.Navigate().GoToUrl("https://demo.seleniumeasy.com/dynamic-data-loading-demo.html");
+
+            //Click "Get New User" button
+            IWebElement getNewUserButton = _driver.FindElement(By.Id("save"), 5);
+            getNewUserButton.Click();
+
+            //Verify "loading..." message is displayed
+            IWebElement loadingMessage = _driver.FindElement(By.Id("loading"));
+            Assert.AreEqual(true, loadingMessage.Displayed);
+
+            //Verify that after 3 seconds of waiting a new user is displayed
```

Comment:
you should interact with driver and elements to detect  when it is loaded

Reply:
Made a new logic to verify new user is displayed.